### DISCUSSION TITLE
Configuring swanctl.conf in other location than /etc/swanctl/swanctl.conf

### DISCUSSION THREAD

**Discussion by gowdamanil** [strongswan/strongswan#2191] [Q&A]
I have configured ipsec using swanctl.conf,
When i use ipsec restart, and then swanctl --load-all, the ipsec starts and works fine.

What i wanted to know is the config file is taken from /etc/swanctl/swanctl.conf when i use ipsec restart or swanctl.conf, but suppose i want it to be picked(while doing ipsec restart and swanctl --load-all) from other location(let us say /tmp/swanctl.conf for just example), how to do this?
Can u suggest the config changes?

Regards,
Anil Gowda M

**Comment by tobiasbrunner** [comment]
[See the documentation](https://docs.strongswan.org/docs/5.9/swanctl/swanctlDir.html) for options to specify different config files/directories for swanctl.
